datafusion/functions-table/src/generate_series.rs

```diff
@@ -197,11 +197,17 @@ impl TableFunctionImpl for GenerateSeriesFuncImpl {
         }
 
         let mut normalize_args = Vec::new();
-        for expr in exprs {
+        for (expr_indice, expr) in exprs.iter().enumerate() {
```

Comment:
I think `index` is a more standard singular form of `indices`

so like

```suggestion
        for (expr_index, expr) in exprs.iter().enumerate() {
```